storybookjs/storybook
# UI: Docs loading state (WIP)

Issue: #16051

## What I did

This is the current state of https://github.com/storybookjs/storybook/pull/16538, cleaned up to be merged to `next`.

Discussed with @MichaelArestad and although he wants to make some more changes to clean things up in that PR, he is happy to merge this for the RC, as what we have so far is a lot better than nothing (what's on `next` at the moment).

## How to test


Yes, see UI in Chromatic

Also, you can experience the loading states when browsing between components and in and out of docs mode in `react-ts`. Not 100% perfect yet but close.
